spec/support/acceptance/servicenow/mock_instance.rb

```diff
@@ -0,0 +1,107 @@
+# rubocop:disable RSpec/HookArgument
+
+require 'json'
+require 'securerandom'
+require 'sinatra'
+
+# Our mock ServiceNow instance is a Sinatra server that mimics
+# the relevant ServiceNow API endpoints used by the tests
+#
+# TODO: See if we can get a persistent dev instance that only CI
```

Comment:
Eh, I guess the mock instance makes stray-CMDB-entry cleanup easier. Also easier if there are concurrent tests running. So this TODO's probably unnecessary.